tools/bin/go_core_loop_mercury

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -o pipefail
+set +e
+
+SCRIPT_PATH=`dirname "$0"`; SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
+OUTPUT_FILE=${OUTPUT_FILE:-"./output.txt"}
+USE_TEE="${USE_TEE:-true}"
+
+echo "Failed tests and panics: ---------------------"
+echo ""
+GO_LDFLAGS=$(bash tools/bin/ldflags)
+use_tee() {
+  if [ "$USE_TEE" = "true" ]; then
+    tee "$@"
+  else
+    cat > "$@"
+  fi
+}
+
+export CL_MERCURY_CMD=`eval "which chainlink-mercury"`
+
+go test -json -ldflags "$GO_LDFLAGS" $TEST_FLAGS -run="^TestIntegration_MercuryV*" ./core/services/ocr2/plugins/mercury | use_tee $OUTPUT_FILE
```

Comment:
iirc the trailing `*` should not be in the pattern match. i think should be `^TestIntegration_MercuryV`

how do we know if the tests actually ran? i looking the log for this ci run and didn't see anything (perhaps because of the trailing `*`) i couldn't tell if there was no matches or if the logs got eaten

Reply:
https://github.com/smartcontractkit/chainlink/actions/runs/7849624416/job/21423373460?pr=11987
```
Run ./tools/bin/go_core_loop_mercury ./...
  ./tools/bin/go_core_loop_mercury ./...
  shell: /usr/bin/bash -e {0}
  env:
    CL_DATABASE_URL: ***localhost:543[2](https://github.com/smartcontractkit/chainlink/actions/runs/7849621121/job/21423362866?pr=11987#step:15:2)/chainlink_test?sslmode=disable
    PNPM_HOME: /home/runner/setup-pnpm/node_modules/.bin
    PATH: /home/runner/wasmd-build/bin:/home/runner/go/bin:/opt/hostedtoolcache/go/1.21.[3](https://github.com/smartcontractkit/chainlink/actions/runs/7849621121/job/21423362866?pr=11987#step:15:3)/x6[4](https://github.com/smartcontractkit/chainlink/actions/runs/7849621121/job/21423362866?pr=11987#step:15:4)/bin:/opt/hostedtoolcache/node/1[6](https://github.com/smartcontractkit/chainlink/actions/runs/7849621121/job/21423362866?pr=11987#step:15:6).20.2/x64/bin:/home/runner/setup-pnpm/node_modules/.bin:/home/runner/.local/share/solana/install/active_release/bin:/home/runner/go/bin:/opt/hostedtoolcache/go/1.21.3/x64/bin:/opt/hostedtoolcache/node/16.20.2/x64/bin:/home/runner/setup-pnpm/node_modules/.bin:/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
    OUTPUT_FILE: ./output.txt
    USE_TEE: false
Failed tests and panics: ---------------------


----------------------------------------------

Beginning check of output logs for sensitive strings
No matches in test log output against known set of sensitive strings
Exit code: 0
All tests passed!
```

Reply:
Hmmm. There is a lot of data in the `output.txt`. I'll remove the `*` and run again to see what happens.